hack/provision.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# DESCRIPTION:
+# This is a script that bootstraps basic content in the OpenShift server
+# to facilitate development flows and users that want to kick the tires.
+# 
+# PROVISIONS:
+# * Users
+# ** test-cluster-admin
+# ** test-admin
+# * Docker registry
+# * OpenShift router
+# * OpenShift templates
+
+# INSTRUCTIONS:
+# $ git clone ...
+# $ make all
+# $ openshift start
```

Comment:
sorry, wasn't intending to be picky... things will appear to work without sudo but actually won't for real deployment cases...